lms/static/scripts/frontend_apps/components/test/BookPicker-test.js

```diff
@@ -175,12 +185,5 @@ describe('BookPicker', () => {
     );
     assert.equal(errorDisplay.prop('error'), error);
     assert.isFalse(picker.exists('ChapterList'));
-
-    // The modal content should not have the class applied for showing the
-    // list of chapters. That class makes the modal too tall for comfort when
-    // displaying an error
-    const modalContent = picker.find('div[role="dialog"]');
-    assert.isTrue(modalContent.hasClass('LMS-Dialog--wide'));
-    assert.isFalse(modalContent.hasClass('LMS-Dialog--tall'));
```

Comment:
These tests no longer apply thanks to the more simplified application of height styling.